apps/crn-server/test/fixtures/users.fixtures.ts

```diff
@@ -154,6 +155,19 @@ export const getGraphQLUser = (
           },
         ],
       },
+      {
```

Comment:
to be honest with you i don't think you should add another team just to test one property (which is essentially what you are doing). Why don't you simply write a single test where the team has the `inactiveSinceDate` property instead?

Reply:
sorry my intention was to reproduce a user that is part of two teams and is active in one and inactive in the other one

Reply:
cool, so then remove this second team :D